go.mod

```diff
@@ -3,6 +3,10 @@ module github.com/denisenkom/go-mssqldb
 go 1.11
 
 require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/golang-sql/civil v0.0.0-20190719163853-cb61b32ac6fe
+	github.com/stretchr/testify v1.7.0
+	github.com/swisscom/mssql-always-encrypted v0.1.3
```

Comment:
This is a non-starter.

If you want to bring this into this repo for review, we can do that. But we won't be adding an external repo like this.

Reply:
Makes sense, but I wanted to keep the two things separated to not clog too much this repo, especially since one package is feature complete (encryption /+ decryption) but here we only support encryption for now.

Reply:
It would belong in an internal/ package, but not a different repo.